src/cnv/ugbio_cnv/convert_combined_cnv_results_to_output_formats.py

```diff
@@ -0,0 +1,456 @@
+import argparse
+import logging
+import statistics
+import subprocess
+import sys
+import warnings
+from os.path import join as pjoin
+
+import numpy as np
+import pandas as pd
+import pysam
+from ugbio_core.logger import logger
+
+warnings.filterwarnings("ignore")
+
+# NOTE: both the id and the column name should appear in the registry
+# (after converting table to BED the names are as in the VCF)
+INFO_TAG_REGISTRY = {
+    "CNV_calls_source": (
+        "CNV_SOURCE",
+        1,
+        "String",
+        "the tool called this CNV. can be combination of: cn.mops, cnvpytor, gridss",
+        "INFO",
+    ),
+    "CNV_SOURCE": (
+        "CNV_SOURCE",
+        1,
+        "String",
+        "the tool called this CNV. can be combination of: cn.mops, cnvpytor, gridss",
+        "INFO",
+    ),
+    "jalign_written": ("JUMP_ALIGNMENTS", 1, "Float", "Number of jump alignments supporting this CNV", "INFO"),
+    "JUMP_ALIGNMENTS": ("JUMP_ALIGNMENTS", 1, "Float", "Number of jump alignments supporting this CNV", "INFO"),
+}
+
+FILTER_COLUMNS_REGISTRY = ["jalign_filter", "LCR_label_value"]
+JALIGN_FILTER_VALUE = "NO_JUMP_ALIGNMENT"
+FILTER_TAG_REGISTRY = {
+    "Clusters": ("Clusters", None, None, "Overlaps with locations with frequent clusters of CNV", "FILTER"),
+    "Coverage-Mappability": (
+        "Coverage-Mappability",
+        None,
+        None,
+        "Overlaps with low coverage or low mappability regions",
+        "FILTER",
+    ),
+    "Telomere_Centromere": (
+        "Telomere_Centromere",
+        None,
+        None,
+        "Overlaps with telomere or centromere regions",
+        "FILTER",
+    ),
+    JALIGN_FILTER_VALUE: ("NO_JUMP_ALIGNMENT", None, None, "No jump alignment support", "FILTER"),
```

Comment:
if you tend to release this without the jump-alignments validation for DUP calls, need to be declared that this is relevant for DEL only.